test/test_mypy.py

```diff
@@ -14,22 +14,17 @@
 
 """Test that each file in mypy_fails/ actually fails mypy, and test some
 sample client code that uses PyMongo typings."""
-
 import os
 import tempfile
 import unittest
 from typing import TYPE_CHECKING, Any, Dict, Iterable, Iterator, List
 
-try:
```

Comment:
We need to keep this in a try/except so that `python test/test_mypy.py` works even when typing_extensions is not installed.